lib/solarwinds_apm/config.rb

```diff
@@ -75,6 +76,28 @@ def self.set_log_level
       SolarWindsAPM.logger.level = debug_level < 0 ? 6 : [4 - debug_level, 0].max
     end
 
+    def self.enable_disable_config(env_var, key, value)
+      default_value = :enabled # Set a default value
+
+      if !env_var.nil? && %w[enabled disabled].include?(ENV[env_var].to_s.downcase)
+        value = ENV[env_var].downcase.to_sym
+      elsif !env_var.nil? && !ENV[env_var].to_s.empty?
+        SolarWindsAPM.logger.warn("[#{name}/#{__method__}] :#{env_var} must be :enabled/:disabled (current setting is #{ENV[env_var]}. Using default value.")
```

Comment:
AFAIK env var values are always string, so for an option like trigger trace (and tag sql) we should expect:
* env var value is string like `enabled`, `true`
* config file value is symbol like `:enabled`, or boolean like true.

```suggestion
        SolarWindsAPM.logger.warn("[#{name}/#{__method__}] #{env_var} must be enabled/disabled (current setting is #{ENV[env_var]}. Using default value.")
```